base/src/main/java/com/smartdevicelink/proxy/rpc/GetVehicleData.java

```diff
@@ -1,793 +1,854 @@
-/*

```

Comment:
@kostyaBoss It looks like you ran the generator tool and re-generated the entire file.  Can you just merge in the changes needed for this PR (external Temperature, climate Data)?